compiler/rustc_middle/src/middle/codegen_fn_attrs.rs

```diff
@@ -55,8 +55,8 @@ pub struct CodegenFnAttrs {
 pub struct TargetFeature {
     /// The name of the target feature (e.g. "avx")
     pub name: Symbol,
-    /// The feature is implied by another feature, rather than explicitly added by the
-    /// `#[target_feature]` attribute
+    /// The feature is implied by another feature or by a witness argument, rather than explicitly
```

Comment:
Calling these witnesses is a bit confusing given their other usage in the compiler lol